### ISSUE TITLE
When using Fluentlenium 3.7.0 getCapabilities() does not work.

### ISSUE THREAD

**Issue by salvada** [FluentLenium/FluentLenium#629]
When using getCapabilities with 3.7.0 I get an error that claims that the object cannot be converted to JSON object.
When trying to debug I noticed that there was a change in PropertiesBackendConfiguration.java

BEFORE: return jsonConverter.toType(property, DesiredCapabilities.class);
NOW: return (Capabilities)this.jsonConverter.toType(property, DesiredCapabilities.class);

**Comment by koryl** [comment]
Thank you for this bug report! There was some changes in PropertiesBackendConfiguration related to changes in Selenium itself and we will modify this in the next milestone.

**Comment by slawekradzyminski** [comment]
Have you tried capabilities() method instead of getCapabilities()? In our tests using FluentLenium 4.0.0 getCapabilites() is null but capabilities() work well.

` @Test
    public void example() {
        Capabilities capabilities = capabilities();
        Capabilities getCapabilities = getCapabilities();
        assertThat(capabilities).isNotNull();
        assertThat(getCapabilities).isNotNull();
    }`

**Comment by slawekradzyminski** [comment]
@salvada I'm not sure if that's the bug. Can you please paste us your @FluentConfiguration line? Assuming you were using headless Chrome here is how it should look

```@FluentConfiguration(webDriver = "chrome", capabilities = "{\"chromeOptions\": {\"args\": [\"headless\",\"disable-gpu\"]}}")

**Comment by salvada** [comment]
Hi all,

yes I tried with 3.7.1 to use capabilities() and i get null, while with the
getCapabilities() the problem reported in the issue is still there.
So both cases don't work for me.
I am using 3.6.1 fluentlenium version that works still fine for me.
You have closed the issue, do you have other suggestion to fix this
capabilities problem?

P.S:
I tried to use also fluentlenium 4.0.0 but I get another problem:
Error:(3, 39) java: cannot access
org.fluentlenium.assertj.FluentLeniumAssertions
  bad class file:
C:\Users\sauria\.m2\repository\org\fluentlenium\fluentlenium-assertj\4.0.0\fluentlenium-assertj-4.0.0.jar(org/fluentlenium/assertj/FluentLeniumAssertions.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of
the classpath.

I am still using java 8 so it could be this.




Il giorno ven 28 dic 2018 alle ore 07:42 Filip Cynarski <
notifications@github.com> ha scritto:

**Comment by puresk** [comment]
No info about this?

**Comment by filipcynarski** [comment]
Hello @puresk 

In terms of 4.0.0 it is for Java 11 so that's why you face the issues as you described.

For FL versions 3.7.x you haven't replied to @slawekradzyminski question from Dec 21, 2018, could you please take a look on that and provide more data to help us to help you.

**Comment by puresk** [comment]
Hi, yes i found that out. I had problem with Actions and that leads me to upgrade Fluentlenium. But it was actually problem with FindBy annotation which i mentioned in #925 . So thanks for reply.

**Comment by salvada** [comment]
Yes the problem it appears also with 3.9.0, that's why until now i was using 3.6.1 because a;ll versions later give me this problem.

getCapabilities gives me still the same problem.
 org.fluentlenium.configuration.ConfigurationException: Can't convert JSON Capabilities to Object.
at org.fluentlenium.configuration.CapabilitiesConfigurationPropertyRetriever.convertJsonObjectToCapabilities

maybe I do something wrong, if you could enlighten me, I'd appreciate it.

thanks
public boolean isInternetExplorer()
{
   return getWebDriver().equals("remote") && getCapabilities().getBrowserName().equals("internet explorer") ||
      getWebDriver().equals("ie");
}

public boolean isFirefox()
{
   return getWebDriver().equals("remote") && getCapabilities().getBrowserName().equals("firefox") ||
      getWebDriver().equals("firefox");
}

**Comment by filipcynarski** [comment]
@salvada As far as I remember Selenium started using theirs own JSON parser that's why you may getting this error. Could you please provide which capabilities are you setting?

**Comment by filipcynarski** [comment]
From version 3.14.0 of Selenium when you are providing capabilities due to the reason which I've explained above you need to do it in different way:

**previously:**
`capabilities = "{javascriptEnabled: true}"`
**NOW**            
`capabilities = "{\"javascriptEnabled\": true}"`

So you have to put JSON key in **quotes**

**Comment by salvada** [comment]
capabilities = "{javascriptEnabled: true}"
This is for default? because I have never set.

This is my fluentlenium properties:
webDriver=remote
pageLoadTimeout=30000
implicitlyWait=3000
awaitAtMost=10000
scriptTimeout=1200000
screenshotPath=target/screenshot
screenshotMode=AUTOMATIC_ON_FAIL
htmlDumpPath=target/screenshot
htmlDumpMode=AUTOMATIC_ON_FAIL


-Dfluentlenium.capabilities="{browserName:\"chrome\"}" and this one is what I set when running

**Comment by salvada** [comment]
capabilities = "{javascriptEnabled: true}"
This is for default? because I have never set.

This is my fluentlenium properties:
webDriver=remote
pageLoadTimeout=30000
implicitlyWait=3000
awaitAtMost=10000
scriptTimeout=1200000
screenshotPath=target/screenshot
screenshotMode=AUTOMATIC_ON_FAIL
htmlDumpPath=target/screenshot
htmlDumpMode=AUTOMATIC_ON_FAIL

-Dfluentlenium.capabilities="{browserName:"chrome"}" and this one is what I
set when running

Il giorno mer 14 ago 2019 alle ore 15:37 Filip Cynarski <
notifications@github.com> ha scritto:
